pyproject.toml

```diff
@@ -24,7 +24,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
```

Comment:
Why though?

Reply:
```
 The current project's Python requirement (>=3.6,<4.0) is not compatible with some of the required packages Python requirement:
    - anyio requires Python >=3.6.2, so it will not be satisfied for Python >=3.6,<3.6.2
```

Reply:
wooh, poetry dependency resolver is great...